ctrl+z (undo) doesn't work after save anymore since 1.5

There's no way of reverting a change that was made if the file was saved. I don't know if that's a feature (saved=saved), but it's really annoying, especially when working with git. I had to reinstall 1.4 because of that, I cannot do any work without my "undo"^^

Windows 10 x86_64
Ubuntu 15.10 x64

**EDIT:** november 1st, 2016 : TL&DR; check out https://github.com/adobe/brackets/issues/11826#issuecomment-257429242

I think this is what you meant:

Download the React DevTools for a better development experience: http://fb.me/react-devtools

/utils/DeprecationWarning.js:90 PreferencesManager.convertPreferences() has been deprecated. Please upgrade to the current Preferences system (https://github.com/adobe/brackets/wiki/Preferences-System#conversion-from-the-pre-36-preferences-system).
    at Object.convertPreferences (/preferences/PreferencesManager.js:325:28)
    at Object.eval (/utils/Resizer.js:548:24)
    at Object.context.execCb (file:///C:/Program%20Files%20(x86)

/command/Menus.js:820 Error setting menu item shortcut: 1
/command/Menus.js:820 (anonymous function)
/LiveDevelopment/MultiBrowserImpl/transports/NodeSocketTransport.js:87 NodeSocketTransport - start

file:///C:/Program%20Files%20(x86)/Brackets/www/SELECT%20*%20FROM%20custome…=%20:address%20AND%20postalcode%20=%20:postalcode%20AND%20city%20=%20:city Failed to load resource: net::ERR_FILE_NOT_FOUND

--- a/src/filesystem/FileSystem.js
+++ b/src/filesystem/FileSystem.js
@@ -817,7 +817,7 @@ define(function (require, exports, module) {
             var oldStat = entry._stat;
             if (entry.isFile) {
                 // Update stat and clear contents, but only if out of date
-                if (!(stat && oldStat && stat.mtime.getTime() === oldStat.mtime.getTime())) {
+                if (!(stat && oldStat && stat.mtime.getTime() <= oldStat.mtime.getTime())) {
                     entry._clearCachedData();
                     entry._stat = stat;
                     this._fireChangeEvent(entry);

